docs/_pages/getting-started.md

```diff
@@ -0,0 +1,155 @@
+---
+permalink: /getting-started
+title: Getting Started
+toc: true
+---
+
+To get started with Luau you need to install Roblox Studio, which you can download [here](https://www.roblox.com/create). 
+
+## Creating a place
+
+If you just want to experiment with the language itself, you can create a simple baseplate game.
+
+<figure>
+  <img src="{{ site.url }}{{ site.baseurl }}/assets/images/create-new-place.png">
+</figure>
+
+## Creating a script
+
+To create your own testing script, go to ServerScriptService in the explorer tree and add a Script object.
+
+<figure>
+  <img src="{{ site.url }}{{ site.baseurl }}/assets/images/create-script.png">
+</figure>
+
+Double-click on the script object and paste this:
+
+```lua
+function ispositive(x)
+    return x > 0
+end
+
+print(ispositive(1))
+print(ispositive("2"))
+
+function isfoo(a)
+    return a == "foo"
+end
+
+print(isfoo("bar"))
+print(isfoo(1))
+```
+
+Note that there are no warnings about calling ``ispositive()`` with a string, or calling ``isfoo()`` a number. 
+
+## Enabling type inference
+
+Now modify the script to include ``--!strict`` at the top:
```

Comment:
This section title is a bit misleading, as type inference is enabled even in non-strict mode (although our default is still noinfer, it will change in the future); strict mode merely makes the inference more aggressive and the type checking - rigorous.